airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/formats/csv_parser.py

```diff
@@ -137,7 +146,7 @@ def _get_schema_dict_without_inference(self, file):
         field_names = next(reader)
         return {field_name.strip(): pyarrow.string() for field_name in field_names}
 
-    def stream_records(self, file: Union[TextIO, BinaryIO]) -> Iterator[Mapping[str, Any]]:
+    def stream_records(self, file: Union[TextIO, BinaryIO], file_info: FileInfo) -> Iterator[Mapping[str, Any]]:
```

Comment:
why we need `file_info` param in `stream_records` method  ?
I see it's not used

Reply:
Agree. It was used for files' chunking. But I removed this logic. 
Updated